homeassistant/components/ituran/device_tracker.py

```diff
@@ -0,0 +1,68 @@
+"""Device tracker for Ituran vehicles."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from pyituran import Vehicle
+
+from homeassistant.components.device_tracker import TrackerEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import IturanConfigEntry
+from .const import ATTR_ADDRESS, ATTR_HEADING, ATTR_LAST_UPDATE
+from .coordinator import IturanDataUpdateCoordinator
+from .entity import IturanBaseEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: IturanConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Ituran tracker from config entry."""
+    coordinator = config_entry.runtime_data.coordinator
+    entities: list[IturanDeviceTracker] = [
+        IturanDeviceTracker(coordinator, vehicle)
+        for vehicle in coordinator.data.values()
+    ]
+    async_add_entities(entities)
+
+
+class IturanDeviceTracker(IturanBaseEntity, TrackerEntity):
+    """Ituran device tracker."""
+
+    _attr_force_update = False
+    _attr_translation_key = "car"
+    _attr_icon = "mdi:car"
+
+    def __init__(
+        self,
+        coordinator: IturanDataUpdateCoordinator,
+        vehicle: Vehicle,
+    ) -> None:
+        """Initialize the Tracker."""
+        super().__init__(coordinator, vehicle)
+
+        self._attr_unique_id = vehicle.license_plate
+        self._attr_name = None
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return entity specific state attributes."""
+        return {
+            ATTR_ADDRESS: self.vehicle.address,
+            ATTR_HEADING: self.vehicle.heading,
+            ATTR_LAST_UPDATE: self.vehicle.last_update,
+        }
```

Comment:
Let's remove these extra state attributes, I think they can be separate sensors instead (but for a separate PR)